.gitignore

```diff
@@ -41,19 +41,6 @@ buildNumber.properties
 .jekyll-cache/
 _site/
 
-# Python artifacts
-src/main/python/systemds/systemds-java/
```

Comment:
I think it is better only to have one gitignore file for the project.

unless the root .gitignore file is ignoring unintended files in src/main/python!

Reply:
i would tend to agree, but in this case the python part is sort of a separate system, that could be in its own repository, and when i develop in the python i teat it as such, therefore i don't have access to the root .gitignore in the IDE when developing the python.

there are examples of doing both designs in Apache.

Hadoop only does one .gitignore
while Spark has one in each language
https://github.com/apache/spark